lib/news_scraper/trainer/preset.rb

```diff
@@ -0,0 +1,76 @@
+module NewsScraper
+  module Trainer
+    class Preset
+      PROVIDER_PHRASE = 'I will provide a pattern using'
+
+      def initialize(uri:, scrape_patterns:, payload:, data_type_presets:, data_type:)
```

Comment:
You can remove `scrape_patterns` since you don't need to pass it into `Transformers::Article` (see below comment).